opsdroid/database/postgresql/__init__.py

```diff
@@ -27,7 +27,10 @@ async def wrapper(*args, **kwargs):
         async with connection.transaction():
             # Create table if it does not exist
             await connection.execute(
-                'CREATE TABLE IF NOT EXISTS "{}" ( key text PRIMARY KEY, data text)'.format(table_name)
+                'CREATE TABLE IF NOT EXISTS "{}" ( key text PRIMARY KEY, data JSONb)'.format(table_name)
+            )
+            await connection.execute(
+                'CREATE INDEX IF NOT EXISTS idxgin ON "{}" USING gin (data);'.format(table_name)
```

Comment:
Why do we want JSONb and INDEX?

Reply:
So we can't assume that this is the only application accessing the data. In fact, when I personally use this, I will be hooking up the database to a [Hasura GraphQL](https://hasura.io/) application. If you use an indexed jsonb field, you can write queries with `WHERE` clauses based on the data in the jsonb field. You can't do that with a text field. This makes it infinitely more useful 👍 (https://medium.com/hackernoon/how-to-query-jsonb-beginner-sheet-cheat-4da3aa5082a3)